Add Java example for AAS Linux Container (#18998)
DataDog/documentation: content/en/serverless/azure_app_services/azure_app_services_container.md

[1]: https://github.com/DataDog/serverless-self-monitoring/tree/main/self_monitor/azure/App_Service_Linux/container/go
[2]: /tracing/trace_collection/dd_libraries/go/?tab=containers#instrument-your-application

{{< /programming-lang >}}
{{< programming-lang lang="java" >}}

Instrument your application with the Datadog Agent by adding the following lines to your Dockerfile. You may need to adjust these examples depending on your existing Dockerfile setup.

```
# copy the Datadog `serverless-init` into your Docker image
COPY --from=datadog/serverless-init /datadog-init /app/datadog-init

# install the java tracing library
ADD https://dtdg.co/latest-java-tracer dd-java-agent.jar

# change the entrypoint to wrap your application into the Datadog serverless-init process
ENTRYPOINT ["/app/datadog-init"]

# execute your binary application wrapped in the entrypoint
CMD ["java", "-javaagent:dd-java-agent.jar", "-jar", "app.jar"]
```

#### Install tracing library

> Tracing should work with the Dockerfile in the previous step. You can alternatively follow [these instructions][2] to install and configure the Java tracing library in your application to capture and submit traces.

Reviewer: ```suggestion
The Dockerfile in the previous step installs and configures the Java tracing library in your application to capture and submit traces.
Tracing should work with the Dockerfile in the previous step. Alternatively, see [Tracing Java Applications - Instrument your application][2].
```
